.github/workflows/karma.yml

```diff
@@ -144,3 +144,9 @@ jobs:
           path: ./packages/@lwc/integration-karma/coverage
 
       - run: yarn coverage
+
+      - name: Upload combined coverage
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage-report-combined
+          path: ./packages/@lwc/integration-karma/coverage/combined
```

Comment:
We might as well upload the combined coverage files, especially now that the HTML actually works.